microprofile/metrics/src/main/java/io/helidon/microprofile/metrics/MetricsCdiExtension.java

```diff
@@ -91,14 +102,35 @@
     private static final Logger LOGGER = Logger.getLogger(MetricsCdiExtension.class.getName());
 
     private static final List<Class<? extends Annotation>> METRIC_ANNOTATIONS
-            = Arrays.asList(Counted.class, Metered.class, Timed.class, Gauge.class, ConcurrentGauge.class);
+            = Arrays.asList(Counted.class, Metered.class, Timed.class, Gauge.class, ConcurrentGauge.class,
+                            SimplyTimed.class);
+
+    private static final List<Class<? extends Annotation>> JAX_RS_ANNOTATIONS
```

Comment:
This is probably good enough.  Note that strictly speaking a JAX-RS annotation is one that is, in turn, annotated with [`HttpMethod`](https://jakarta.ee/specifications/restful-ws/2.1/apidocs/javax/ws/rs/HttpMethod.html).  I'd probably leave this line alone anyway.

Reply:
I want to make sure I understand. 

I think you're suggesting that instead of iterating through the individual annotation classes {`GET.class, etc.) to see if one is present on a given method we could look for `HttpMethod.class`. Is that it?

Related, the `@Observes @WithAnnotations` clause currently lists the individual JAX-RS-related annotations (`GET.class`, etc.) in the method declaration. Could we also use  `HttpMethod.class` there?